src/pseudopeople/interface.py

```diff
@@ -447,9 +448,9 @@ def generate_taxes_1040(
     :param seed: An integer seed for randomness. Defaults to 0.
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
-    :param year: The tax year (format YYYY) to include in the dataset. Will raise
-        a ValueError if there is no data with this year. If None is provided,
-        data from all years are included in the dataset.
+    :param year: The tax year for which data was collected (format YYYY, e.g., 2036). Will raise
```

Comment:
"The tax year about which taxes were filed"? Although, see my comment above about the past tense.

Reply:
Yeah, I thought past tense sounded kind of weird, but I wasn't sure what else to say. I copied the description from the "tax_year" column description on the Datasets page.

Reply:
I changed the wording here and in all the other descriptions to say something along the lines of "The year for which to generate data," but with specific details for each dataset. Let me know if the new version sounds good.